recOrder/plugin/workers/calibration_workers.py

```diff
@@ -12,6 +15,13 @@
 import logging
 import json
 
+# type hint/check
+from typing import TYPE_CHECKING
+# avoid circular import error
+if TYPE_CHECKING:
+    from recOrder.plugin.widget.main_widget import MainWidget
+    from recOrder.calib.Calibration import QLIPP_Calibration
+
```

Comment:
Can you help me understand this?

Reply:
`main_widget` imports from `calibration_worker` and causes circular import issues in the base-class declaration. This avoids errors by only executing during type checking calls, which currently never happen in runtime. This is both for readability (native type hints) and being future-proof (if we will be using `pydantic` in 1.0.0+).